src/main/java/com/es/phoneshop/web/ProductListPageServlet.java

```diff
@@ -56,4 +82,52 @@ private ProductSortCriteria getSortCriteria(HttpServletRequest request) {
                 .build() : null;
     }
 
+    private boolean isProductAddedToCart(HttpServletRequest request, HttpServletResponse response, UUID id)
```

Comment:
i know about code duplication from PDP servlet here
want to discuss where it is better to move these methods